ushahidi/platform-client
# Feature/metadata screen

This pull request makes the following changes:
- Adds ui to add metadata for a hxd-upload
- Fetches licenses and organisations from api
- Saves metadata to api

Testing checklist:
- go to data-export->Assign tags and attributes, assign a number of fields with tags and attributes
- Click on upload to hdx
- [ ] You should be redirected to the metadata-screen
- Fill in the required fields
- [ ] There should be data in the organisations-dropdown
- [ ] There should be data in the licenses-dropdown
- Click on 'upload data'
- [ ] Upload should start
- [ ] You should see a notification-window
- [ ] The in progress-screen should be visible


Fixes ushahidi/platform# .

Ping @ushahidi/platform
